Mathlib/Algebra/Group/Aut.lean

```diff
@@ -37,8 +37,6 @@ def MulAut (M : Type*) [Mul M] :=
 #align mul_aut MulAut
 #align add_aut AddAut
 
-attribute [reducible] AddAut
```

Comment:
I suspect this one is still needed?

Reply:
Okay, I agree this one needs to be changed.